src/components/input-field/input-field.scss

```diff
@@ -20,6 +20,15 @@
     display: none;
 }
 
+:host([type='textarea']) {
+    height: inherit; // This makes it possible for textarea in limel-form to stretch & span through rows
+
+    .mdc-text-field.mdc-text-field--textarea {
+        height: var(--textarea-height, 100%);
+        resize: vertical;
+    }
```

Comment:
This isn't working as expected when used in limel-form 🙁

See the following 2 gifs.

Multi-column:

![textarea in limel-form 1](https://user-images.githubusercontent.com/154725/100341548-f7b3d300-2fdc-11eb-861b-c9d68bc08fe3.gif)

Single-column:

![textarea in limel-form 2](https://user-images.githubusercontent.com/154725/100341578-026e6800-2fdd-11eb-93c9-cc954c84bba5.gif)

Reply:
Bump on this.

@Kiarokh